src/Jackett.Common/Definitions/carpt.yml

```diff
@@ -72,14 +72,9 @@ login:
     selector: img[alt="CAPTCHA"]
     input: imagestring
   inputs:
-    secret: ""
     username: "{{ .Config.username }}"
```

Comment:
the secret keyword is hidden and is in the payload sent by the browser so this should not be removed from the indexer
```
secret=&username=redacted&password=redacted&two_step_code=&imagestring=redacted&imagehash=redacted
```